vyper/parser/stmt.py

```diff
@@ -114,7 +113,7 @@ def assign(self):
         sub = Expr(self.stmt.value, self.context).lll_node
         # Determine if it's an RLPList assignment.
         if isinstance(self.stmt.value, ast.Call) and getattr(self.stmt.value.func, 'id', '') is 'RLPList':
-            pos = self.context.new_variable(self.stmt.targets[0].id, set_default_units(sub.typ))
+            pos = self.context.new_variable(self.stmt.targets[0].id, sub.typ)
```

Comment:
Does `set_default_units` happen automagically now?

Reply:
Or wait, the function doesn't exist anymore, right?

Reply:
Correct that's the one I mentioned on #654 ;) where it all boils down to return typ